Packs/Azure-Enrichment-Remediation/Playbooks/Azure_-_Network_Security_Group_Remediation.yml

```diff
@@ -584,6 +596,15 @@ tasks:
                       value:
                         simple: ${inputs.AzureVMPrivateIP}
                       iscontext: true
+                  - operator: containsGeneral
+                    left:
+                      value:
+                        simple: AzureNSG.Rule.destinationAddressPrefixes
+                      iscontext: true
+                    right:
+                      value:
+                        simple: ${inputs.AzureVMPrivateIP}
```

Comment:
since this is a context value in condition you don't need to point it with the form of `${<key>}`
just try the suggestion and you will see how it will paste that.